### PR TITLE
Add > Cadence Check > Workspace user activity

### PR DESCRIPTION
**Pull request by RahulSrivastav14**
### Checklist
- [x] Issue(s) linked
Screenshot 2024-11-13 at 8 33 47 PM

### CODE REVIEW

**Comment by rajlearner17** [comment]
@misraved can you pls expedite this?

**Comment by misraved** [comment]
@RahulSrivastav14 could you please add the missing docs page for this report?

Thanks!!

File: dashboards/workspace/workspace_activity_report.pp [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+              turbot {
+                akas
+              }
+            }
+          }
+        }'
+    GROUP BY
+        g.workspace;
+  EOQ
+}
```
Comment (misraved) [inline dashboards/workspace/workspace_activity_report.pp:91]:
```suggestion
    select
        g.workspace,
        g.workspace as resource,
        case 
            when count(
                case 
                    when (n.notifications ->> 'email') not like '%@turbot.com' 
                    then 1 
                    else null 
                end
            ) = 0 then 'alarm'
            else 'ok'
        end as status,
        case 
            when count(
                case 
                    WHEN (n.notifications ->> 'email') NOT LIKE '%@turbot.com' 
                    THEN 1 
                    ELSE NULL 
                END
            ) = 0 THEN 'Workspace is Inactive. No User Login for 30 Days.'
            ELSE 'Workspace is Active.'
        END AS reason
    FROM
        guardrails_query g
    LEFT JOIN LATERAL
        jsonb_array_elements(g.output -> 'notifications' -> 'items') AS n(notifications) ON TRUE
    WHERE
        g.query = '{
          notifications: resources(
            filter: "resourceTypeId:tmod:@turbot/turbot-iam#/resource/types/profile,tmod:@turbot/turbot-iam#/resource/types/groupProfile,tmod:@turbot/aws-iam#/resource/types/instanceProfile $.lastLoginTimestamp:>=T-30d"
          ) {
            items {
              email: get(path:"email")
              lastLoginTimestamp: get(path: "lastLoginTimestamp")
              trunk {
                title
              }
              turbot {
                akas
              }
            }
          }
        }'
    GROUP BY
        g.workspace;
  EOQ
}
```

File: dashboards/workspace/workspace_activity_report.pp [hunk 2]
```diff
@@ -0,0 +1,164 @@
+benchmark "workspace_user_activity" {
```
Comment (misraved) [inline dashboards/workspace/workspace_activity_report.pp:1]:
```suggestion
benchmark "workspace_activity" {
```

File: dashboards/workspace/workspace_activity_report.pp [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+  title = "Turbot Guardrails Workspace Activity"
+  documentation = file("./dashboards/workspace/docs/workspace_activity_report.md")
```
Comment (misraved) [inline dashboards/workspace/workspace_activity_report.pp:3]:
```suggestion
  documentation = file("./dashboards/workspace/docs/workspace_activity.md")
```

**Pull request merged**